src/object/obj_internal.h

```diff
@@ -240,6 +240,20 @@ ec_bulk_spec_get_skip(int index, struct ec_bulk_spec *skip_list)
 {
 	return skip_list[index].is_skip;
 }
+
+static inline uint32_t
+ec_rs2cs(uint64_t rs, unsigned char cs_exp)
```

Comment:
is this func to calculate cell size form record size?  some comments about the func and parameters is helpful, thx